resources/translations/en.edn

```diff
@@ -40,7 +40,6 @@
             :remove-member "Remove member"
             :request-decision "Request decision"
             :request-review "Request review"
-            :request-selection "Send request to"
```

Comment:
But this singular is used for request decision as that is always to a single person?